src/frontends/tensorflow/docs/contribute.md

```diff
@@ -0,0 +1,71 @@
+# How to implement support of a new TensorFlow operation
+
+The conversion of a TensorFlow operation requires either one pass using [Loaders](../src/op) or two transformation passes
+using [Loaders](../src/op) and [Internal Transformation](../src/helper_transforms). It is often sufficient to use only [Loaders](../src/op) for the conversion.
```

Comment:
The same sentence as in previous two files

Reply:
I left it only in `contribute.md` now